Library/Homebrew/test/cmd/list_spec.rb

```diff
@@ -4,19 +4,47 @@
 require "cmd/shared_examples/args_parse"
 
 RSpec.describe Homebrew::Cmd::List do
-  let(:formulae) { %w[bar foo qux] }
+  let(:formulae) { %w[bar qux foo] }
+  let(:casks) { %w[tex git sgb] }
 
   it_behaves_like "parseable arguments"
 
   it "prints all installed Formulae", :integration_test do
```

Comment:
We deliberately want to minimise the number of `integration_test` we have because they are very slow. Let's not add any more for `brew list`, please. I'm game to have unit tests for the logic you're testing here.

Reply:
OK. I don't have enough ressources do proceed at this time.